src/Controller/ContactController.php

```diff
@@ -4,17 +4,119 @@
 
 namespace App\Controller;
 
+use App\Constants\Constants;
+use Psr\Log\LoggerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\Request;
+use App\Contact\DataValidatorInterface;
+use App\Contact\DataValidatorContact;
 
+/**
+ * ContactController
+ */
 class ContactController extends AbstractController
 {
+    /**
+     * logger
+     *
+     * @var mixed
+     */
+    private $logger;
+
+    /**
+     * name
+     *
+     * @var string
+     */
+    public $nameContact = 'Georgiana Panaete';
+
+    /**
+     * name
+     *
+     * @var string
+     */
+    public $name = '';
+
+    /**
+     * name
+     *
+     * @var string
+     */
+    public $email = '';
+
+    /**
+     * name
+     *
+     * @var string
+     */
+    public $description = '';
+
+    /**
+     * message
+     *
+     * @var string
+     */
+    public $errorMessage = '';
+
+    /**
+     * validator
+     *
+     * @var mixed
+     */
+    public $validator;
+
+    /**
+     * __construct
+     *
+     * @param  mixed $logger
+     * @return void
+     */
+    public function __construct(LoggerInterface $logger, DataValidatorContact $validator)
```

Comment:
The constructor Dependency Injection uses the interface type-hinting. Use DataValidatorInterface in the constructor instead of DataValidatorContact to be able to change the validator in the future from the configuration files.